<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitSelectionWidget</class>
 <widget class="QWidget" name="UnitSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>423</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Recruit</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">UnitSelectionWidget{
	
	background-image: url(:/image/bricks_background.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Recruit Units</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="3">
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #d1bead;
    border: 3px solid #f4f6e9;
}
QListWidget::item {
    font-size: 10pt; 
    color: #3d4265;
    border: 3px solid #f4f6e9;
    background-color: #dfc686;
    margin: 2px 2px;
}
QListWidget::item:selected {
    color: #f4f4e8;
    background-color: #a47750;
}
QListWidget::item:hover {
    color: #3d4265;
    background-color: #bd9877;
}
</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">margin: 5px 20px;</string>
         </property>
         <property name="text">
          <string>Current Coins</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/image/icon/coin</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="currentCoinsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">margin: 5px 20px</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QDialogButtonBox&gt;QPushButton{
color: #3d4265;
border-radius: 10px;
border: 3px solid #f4f6e9;
background-color: #dfc686;
width: 100px;
padding: 10px 2px;
}
QDialogButtonBox&gt;QPushButton:selected {
    color: #f4f4e8;
    background-color: #a47750;
}
QDialogButtonBox&gt;QPushButton:hover {
    color: #3d4265;
    background-color: #bd9877;
}
QDialogButtonBox&gt;QPushButton:pressed {
    color: #3d4265;
    background-color: rgb(157, 126, 99);
}</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
